package.json

```diff
@@ -45,14 +45,14 @@
     "babel-preset-react": "^6.16.0",
     "classnames": "^2.2.5",
     "compare-sets": "^1.0.1",
-    "core-decorators": "^0.19.0",
-    "diff": "3.2.0",
+    "core-decorators": "^0.20.0",
+    "diff": "3.3.1",
     "dugite": "^1.38.0",
     "etch": "^0.12.4",
     "event-kit": "^2.3.0",
-    "fs-extra": "^2.1.2",
-    "graphql": "^0.10.3",
-    "hoist-non-react-statics": "^1.2.0",
+    "fs-extra": "^4.0.2",
```

Comment:
Fun bonus: fs-extra > 3.0.0 automatically offers [promisified versions of all fs calls](https://www.npmjs.com/package/fs-extra#sync-vs-async) if you omit the callback argument. We should be able to ✂️ our case-by-case helper shims :tada: